<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainDialog</class>
 <widget class="QDialog" name="QMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton_train">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>520</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_test">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Set HyperParameters</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>n_workers</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPU0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Learning Rate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPU1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>GPUs (experimental)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_epochs">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Epochs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPU3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPU2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU 2</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_imgsize">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1024</number>
    </property>
    <property name="value">
     <number>224</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Size</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_lr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="singleStep">
     <double>0.000010000000000</double>
    </property>
    <property name="value">
     <double>0.000100000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Batch Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_batchsize">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1024</number>
    </property>
    <property name="value">
     <number>32</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPU5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU 5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPU4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPU6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU 6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GPU7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU 7</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_device">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nworkers">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1024</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Train Set Path</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pushButton_trainpath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Test Set Path</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pushButton_testpath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
